blackbox/docs/admin/system-information.rst

```diff
@@ -55,6 +55,32 @@ The result has at most 1 row::
   SELECT 1 row in set (... sec)
 
 
+.. _sys-cluster-license:
+
+Cluster License
+---------------
+
+The ``sys.cluster.license`` expression returns information about the currently
+registered cluster license.
+
+::
+
+    cr> select license from sys.cluster;
```

Comment:
I wonder if we should instead have a table that contains the columns + description. The `...` makes this otherwise opaque